pkg/cephmgr/cephleader.go

```diff
@@ -23,6 +23,7 @@ import (
 	etcd "github.com/coreos/etcd/client"
 	"github.com/coreos/etcd/store"
 	"github.com/rook/rook/pkg/cephmgr/client"
+	"github.com/rook/rook/pkg/cephmgr/file"
```

Comment:
not sure I follow why its inconsistent with osd and mon. should osd be called "storage" then? I recommend you just use the ceph role names since this is the cephmgr.